### PR TITLE
Update court-issued-event-codes.csv

### PR DESCRIPTION
**Pull request by klohman**
-removed event code from document type in court-issued list

### CODE REVIEW

**Review by rachaelparris — APPROVED** [review]
For these to actually show up this way in the app, we'll need to run a script to generate the JSON file from the CSV. Do you want to do that in this PR, or is that coming soon?

File: shared/src/tools/court-issued-event-codes.csv [hunk 1]
```diff
@@ -1,57 +1,61 @@
...
+OPX,ORDER time is extended to *DATE* for petr(s) to file Amended Petition.,Order time is extended for petr(s) to file Amended Petition ,Type E,,,[Date],,,Order time is extended to [Date] for petr(s) to file Amended Petition ,Served
+ORAP,ORDER for Amendment to Petition by *DATE*. *ANYTHING*,Order for Amendment to Petition ,Type D,[Anything],,[Date],,,Order for Amendment to Petition by [Date] [Anything],Served
+OROP,ORDER for Ratification of Petition by *DATE*. *ANYTHING*,Order for Ratification of Petition ,Type D,[Anything],,[Date],,,Order for Ratification of Petition by [Date] [Anything],Served
+OSC,ORDER *ANYTHING*,Order to Show Cause ,Type A ,[Anything],,,,,Order [Anything],Served
+OSCP,"ORDER petr(s) by *DATE* show cause why ""S"" should not be removed *ANYTHING*","Order petr(s) to show cause why ""S"" should not be removed ",Type D,[Anything],,[Date],,,"OSCP - Order petr(s) by [Date] to show cause why ""S"" should not be removed [Anything]",Served
```
Comment (rachelschneiderman) [inline shared/src/tools/court-issued-event-codes.csv]: "OSCP - Order petr(s) by [Date] to show cause why ""S"" should not be removed [Anything]" 

Should the 'OSCP -' still be included? I could be totally wrong, just by the other changes seems like all the other event codes were removed?

**Review by rachelschneiderman — APPROVED** [review]

File: shared/src/tools/court-issued-event-codes.csv (hunk 1 above)
Comment (klohman) [reply shared/src/tools/court-issued-event-codes.csv]: nope you're right, i just missed one

File: shared/src/tools/court-issued-event-codes.csv [hunk 2]
```diff
@@ -1,57 +1,61 @@
...
+OST,ORDER of Service of Transcript (Bench Opinion) *ANYTHING*,Order of Service of Transcript (Bench Opinion),Type A ,[Anything],,,,,Order of Service of Transcript (Bench Opinion) [Anything],Served
+OSUB,ORDER that case is submitted to *JUDGE*. *ANYTHING*,Order that case is submitted,Type B,[Anything],[Judge name],,,,Order that case is submitted to [Judge Name] [Anything],Served
+DEC,"DECISION Entered,, *JUDGE* *ANYTHING*","Decision Entered, ",Type B,[Anything],[Judge name],,,,"Decision Entered, [Judge Name] [Anything]",Entered and Served
```
Comment (kkoskelin) [inline shared/src/tools/court-issued-event-codes.csv]: First few occurrences of `DECISION Entered,,` and `"Decision Entered, "` seem to have extra commas?

Comment (klohman) [reply shared/src/tools/court-issued-event-codes.csv]: Fixed! thank you!

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**